public/index.html

```diff
@@ -4,7 +4,7 @@
       <title>Logistration | edX</title>
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
-      <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+      <link rel="shortcut icon" href="<%=htmlWebpackPlugin.options.FAVICON_URL%>" type="image/x-icon" />
   </head>
```

Comment:
What is the intent here?

Reply:
@adeelkhan As we are moving towards a new logo, this will allow us to use the edX defined favicon in remote config.